pages/learn/index.page.tsx

```diff
@@ -39,7 +37,7 @@ export default function Welcome() {
           link='https://tour.json-schema.org/'
         />
       </div>
-      <DocsHelp markdownFile={markdownFile} />
+      <DocsHelp />
```

Comment:
Why this change?

Reply:
Oh, I think it is from one of the previous PR, where there was a redirect to a wrong page. I will correct this asap and will update the PR.

Reply:
Actually , this was part of some other PR , that solves redirecting issue , and by mistake it got reflected in this PR too.
I have rectified the changes. Kindly review the PR and suggest some changes if applicable.